bitnami/harbor/templates/core/core-svc.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: "{{ template "harbor.core" . }}"
+  labels: {{ include "harbor.labels" . | nindent 4 }}
+spec:
+  {{- if  (eq .Values.service.ingress.controller "gce") }}
+  type: NodePort
+  {{- end }}
+  ports:
+    - port: 80
```

Comment:
Shouldn't this port be parametrised?

Reply:
Not really. The service used to access Harbor is the portal one and that is parametrized.